client/app/account/user/agreements/user-agreements.service.js

```diff
@@ -1,19 +1,84 @@
 angular.module("UserAccount").service("UserAccount.services.agreements", [
     "$http",
     "$q",
+    "$translate",
     "UserAccount.constants",
     "$cacheFactory",
-    function ($http, $q, constants, cache) {
+    function ($http, $q, $translate, constants, cache) {
         "use strict";
 
         const userAgreementsCache = cache("USER_AGREEMENTS");
 
         const proxyPath = `${constants.swsProxyRootPath}me`;
 
+        const GDPR_AGREEMENTS_INFOS = [
```

Comment:
You might consider adding a `user-agreements.constants.js` file instead.

Reply:
Also, I've already done something that kind of looks like this with `/me/alerts`, maybe you should check it out to see what you can do about it